src/app/lib/config/services/marathi.js

```diff
@@ -27,10 +27,10 @@ export const service = {
     locale: `mr-IN`,
     datetimeLocale: `mr`,
     service: 'marathi',
-    serviceName: 'News मराठी',
+    serviceName: 'Marathi',
     themeColor: `${C_POSTBOX}`,
     twitterCreator: '@bbcnewsmarathi',
```

Comment:
Does this need to change to `@bbcnews` as well?

Reply:
No - the twitter links are meant to be service-specific (because they are there for audience benefit), whereas the facebook links are all meant to be bbcnews (because they are there for analytics purposes).

Reply:
Ah okay I was just asking because you changed the `twitterSite` to `@bbcnews`https://github.com/bbc/simorgh/blob/83089de6a1d6906128b0380378d21ab219b69434/src/app/lib/config/services/marathi.js#L33

Reply:
Ah. I meant to change it back again. Oops. Will fix Monday. Thanks

Reply:
Sorted,  Thanks